R/create.R

```diff
@@ -359,14 +357,13 @@ create_gp_data <- function(gp = gp_opts(), data) {
   ls_min = gp$ls_min,
   ls_max = data$t - data$seeding_time - data$horizon,
   alpha_sd = gp$alpha_sd,
-  # nolint start
+
```

Comment:
random space in a list?

Reply:
Weird, isn't it? Fixed in bf9459c363c6804400445599532ccb4fa3174296.